File: docs/snippets/elixir/getting-started/extract_with_ocr.exs
```elixir title="Elixir"
# Extract scanned documents with OCR
# Configure Tesseract for OCR processing

ocr_config = %Xberg.Config.OCR{
  backend: "tesseract",
  language: "eng"
}

config = %Xberg.Config.Extraction{
  ocr: ocr_config
}

{:ok, output} = Xberg.extract(%Xberg.ExtractInput{kind: :uri, uri: "scanned.pdf"}, config)

result = List.first(output.results)
IO.puts("Extracted text from scanned document:")
IO.puts(result.content)
IO.puts("Used OCR backend: tesseract")
```

File: docs/snippets/csharp/getting-started/extract_with_ocr.cs
using Xberg;

var config = new ExtractionConfig
{
    ForceOcr = true,
    Ocr = new OcrConfig
    {
        Backend = "tesseract",
        Language = "eng"
    }
};

var result = XbergClient.ExtractSync("scanned.pdf", config);

Console.WriteLine(result.Content);
Console.WriteLine($"Detected Languages: {string.Join(", ", result.DetectedLanguages ?? new List<string>())}");
